.blueprint/generate-sample/generator.mjs

```diff
@@ -0,0 +1,62 @@
+import { readdir } from 'node:fs/promises';
+import BaseGenerator from 'generator-jhipster/generators/base';
+import command from './command.mjs';
+
+export default class extends BaseGenerator {
+    sampleName;
```

Comment:
Should this be 2-space indents instead of 4?

Reply:
prettier config